v3/src/data-model/data-set.ts

```diff
@@ -590,6 +590,14 @@ export const DataSet = types.model("DataSet", {
         })
       },
 
+      setValue(caseID: string, attributeID: string, value:IValueType) {
```

Comment:
Let's leave this for now but I've started working on the refactor we discussed which will change this (e.g. `setCaseValue()`).